resources/views/forms/edit.blade.php

```diff
@@ -26,6 +26,7 @@
     </form>
 
     </table>
+    </head>
```

Comment:
headタグはbodyタグの中身まで囲う必要はないので、以下のように修正お願いいたします！

> head...文書に関する機械可読な情報（メタデータ）、たとえば題名、スクリプト、スタイルシートなどを含むもの

```suggestion
    <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>フォーム一覧</title>
    </head>
```